style.css

```diff
@@ -1 +1,182 @@
-/*  Add your styling here */
+.hamIcon{
+    position: absolute;
+    right: 10px;
+    top: 20px;
+}
+
+.logo_image{
+    display: flex;
+    justify-content: center;
+    background: url("images/cake-logo.jpg");
+    height: 200px;
+    background-position: center;
+}
+.nav_bar{
+    background: url("images/nav_back.JPG");
+    background-repeat: repeat-x;
+    background-size: cover;
+}
+
+.mainMenu{
+    font-family: 'Courgette', cursive;
+    right:2px;
+    top: 100px;
+    padding: 30px;
+    font-size: 30px;
+    color: black;
+    height: 200px;
+    border-radius:10px;
+    border:4px double plum;
+    text-decoration: none;
+    background: url("images/bar-background.jpg");
+
+}
+
+body{
+    background: url("images/wallpaper.jpg");
+}
+
+a:hover{
+    color:blue;
+    text-decoration: none;
+}
+.My_title{
+    color:#fc0384;
+    font-size: 50px;
+    font-family: 'Courgette', cursive;
+    /*display: flex;
+    flex-direction: column;
+    justify-content:center;*/
+    text-align: center;
+    padding: 50px 20px 10px 50px;
+
+}
+.description{
+    font-size: 30px;
+    text-align: center;
+    color: black;
+    font-family: 'Courgette', cursive;
+    padding: 0 auto;
+    margin: 10 10;
+}
+.main_cakes{
+    display:flex;
+    flex-direction:column;
+    border:2px solid pink;
+    border-radius: 10px;
+    border-style: ridge;
+    box-sizing:border-box;
+}
+
+.big_pic{
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    background-position: center;
+    background-repeat: no-repeat;
+    padding: 20px 30px;
+    
+}
+
+.cover1_back{
+    width: 80%;
+    padding: 10px 5px;
+}
+.font_cake{ 
+    font-size:40px;
+    text-align: center;
+    text-decoration: none;
+    color: purple;
+    font-family: 'Courgette', cursive;
+    
+}
+li{
+    list-style-type:none;
+}
+.font_cake2{
+    font-size: medium;
+    text-decoration: none;
+    color: purple;
+    font-family:'Courgette', cursive ;
+    text-align: justify;
+}
+
+.style_cake{
+   display:flex;
+   flex-direction:row;
+   justify-content: space-around;
+   align-items: center;
+}
+
+.width_cakes{
+    width: 50%;
+    height: 50%;
+    text-decoration: none;
+}
+.width_cake3{
+    width: 200px;
+    height: 1%;
```

Comment:
You're setting a width *and* a height here. As we saw in class, you can't set both otherwise it distorts the image. A solution would be to set `width: 100%` on the 3 images (using the *same* class, not different ones), and then give each `li` (=the parent of the `img` tags) a width (in this case, `width: 33%` would work well because there's 3 of them).